.storybook/theme-decorators.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ThemeProvider } from 'styled-components';
+import { withThemesProvider } from 'storybook-addon-styled-component-theme';
+import classic from '../src/style/themes/classic';
+import small from '../src/style/themes/small';
+import medium from '../src/style/themes/medium';
+import large from '../src/style/themes/large';
+import none from '../src/style/themes/none';
+
+// Detects whether the current window is the topmost window in the window hierarchy
```

Comment:
We should use `/** text */` for multiple line comments instead of commenting each line using `//`

Reply:
Updated in 7285f56 :thumbsup: